test/stripe/error_test.exs

```diff
@@ -10,7 +10,10 @@ defmodule Stripe.ErrorTest do
       error = Error.from_stripe_error(400, nil, "id")
       assert error.code == :bad_request
       assert error.extra.http_status == 400
-      assert error.message == "The request was unacceptable, often due to missing a required parameter."
+
+      assert error.message ==
```

Comment:
Maybe name this something like `expected_message`, so you don't break in middle.